### ISSUE TITLE
Create visualizer for the comparisons

### ISSUE THREAD

**Issue by Ameobea** [Ameobea/network_research#27]
Create HTML-based grid visualizer for the results from the comparison script.

Also, change comparison script to output for when there was an error in one of the networks being compared instead of not including it at all.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/118867524991974/137661498055083)

**Comment by Ameobea** [comment]
Looking at various visualization types that exist, I think that a correlogram would be a good choice for the presentation of this data.  

http://www.statmethods.net/advgraphs/correlograms.html

**Comment by Ameobea** [comment]
I've decided upon simpler scatter plots containing correlations between network attributes with each attribute being one axis of the plot.  For example, if exploring correlations between eigenvector centrality and average clustering, the centrality values would be on the X axis and the clustering values would be on the Y axis.  

This seems like a good first step for finding basic correlations but can be expanded upon for more advanced methods.  I've completed raw scatter plots in a5a604e31dbd57c8a0867ec04903c98f23bdfbc0, but a HTML UI is still necessary.
